### ISSUE TITLE
Adapt docs.kalix.io for Spring SDK

### ISSUE THREAD

**Issue by kumrahul16** [lightbend/kalix-jvm-sdk#1337]

**Comment by kumrahul16** [comment]
Did we make any progress on this ?

**Comment by octonato** [comment]
No, actually not. We should plan it, but I wonder if all these changes should be done right now or if we should postpone it until we remove the `beta`. 

We need to decide what are the parts that we still want to change before removing the `beta` from the docs.

**Comment by kumrahul16** [comment]
This is top priority. We can decide if we want to keep or remove 'beta' in parallel. @beritou @octonato  please co-ordinate this thing. We need to start and get this done ASAP.
